### PR TITLE
document our process

### PR DESCRIPTION
**Pull request by 4383**

### CODE REVIEW

File: doc/source/maintenance/process.rst [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+each commits landed in this new version. Here is an example https://github.com/eventlet/eventlet/issues/897.
+
+3. Create the tag
+~~~~~~~~~~~~~~~~~
+
+Once the changelog patch is merged, then we are now
+able to produce the new corresponding tag, here are the
+commands we use to do that:
+
+```bash
```
Comment (itamarst) [inline doc/source/maintenance/process.rst:56]: It's also easy to do this from the GitHub UI.

**Review by itamarst — APPROVED** [review]
Might also want to link this from higher-level docs, or alternatively have it as RELEASING.rst in root or something.

File: doc/source/maintenance/process.rst (hunk 1 above)
Comment (4383) [reply doc/source/maintenance/process.rst:56]: Indeed, I added a sentence related to github UI

**Comment by 4383** [comment]
> Might also want to link this from higher-level docs, or alternatively have it as RELEASING.rst in root or something.

I added a link. I'd prefer to keep all docs things centralized in one place.

**Pull request merged**